test/test_wofpy_soap.py

```diff
@@ -7,7 +7,10 @@
 from examples.flask.swis.swis_dao import SwisDao
 import wof
 import wof.flask
-import ConfigParser
+try:
+    import ConfigParser as configparser
+except ImportError:
+    import configparser
```

Comment:
I always used as a drop-in replacement.We can avoid the `try`/`except` block by installing the backported `configparser`. See http://python-future.org/compatible_idioms.html#configparser

What do you think?